openfe/setup/atommapping.py

```diff
@@ -12,21 +15,133 @@ class AtomMapping:
 
     Attributes
     ----------
-    mol1, mol2 : Molecule
+    mol1, mol2 : rdkit.Chem.rdchem.Mol
       the two Molecules in the mapping
     mol1_to_mol2 : dict
       maps the index of an atom in either molecule **A** or **B** to the other.
       If this atom has no corresponding atom, None is returned.
 
     """
-    mol1: Molecule
-    mol2: Molecule
+
+    mol1: Chem.rdchem.Mol
+    mol2: Chem.rdchem.Mol
```

Comment:
Why is this switching back to RDKit? Current code has `AtomMapper` making an `AtomMapping` using our `Molecule` class. I believe that we NEED this to be the case for analysis to work correctly.

https://github.com/OpenFreeEnergy/openfe/blob/41068e018575a4da5743d1f45e494fbbdc2a9921/openfe/setup/atommapper.py#L40-L62

Reply:
I thought it would need to be an RDKit mol so that things like `bond.GetBeginAtomIdx()` would work in `_draw_mapping`

Reply:
You can get the RDKit version from our Molecule with `mol.rdkit` (or, after #58, `mol.to_rdkit()`). From there you can use RDKit internally.